components/list/list-item-button-mixin.js

```diff
@@ -20,38 +20,15 @@ export const ListItemButtonMixin = superclass => class extends ListItemMixin(sup
 			:host(:not([button-disabled])) {
 				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
 			}
-			:host([button-disabled][_hovering-primary-action]) .d2l-list-item-content,
-			:host([button-disabled][_focusing-primary-action]) .d2l-list-item-content {
-				--d2l-list-item-content-text-color: unset;
-				--d2l-list-item-content-text-decoration: none;
-			}
-			:host([button-disabled]) button {
-				cursor: default;
+			:host([skeleton]) [slot="content-action"],
+			:host([button-disabled]) [slot="content-action"] {
+				pointer-events: none;
 			}
```

Comment:
This doesn't appear to be a regression but it looks like the selection-inputs still work in skeleton mode. 

This can be a separate ticket since it was an issue before as well but I think it would be a good idea to spin up a defect for that.